design/standard/templates/content/datatype/diff/eztags.tpl

```diff
@@ -0,0 +1,16 @@
+{def
+    $old_tags = $diff.old_content.content.keyword_string|explode( '|#' )
```

Comment:
You shouldn't use `keyword_string` here. It's an internal property not intended for usage in templates. You should use `tags` property to get the list of all tags and extract keywords from them with `keyword` property.